aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -1112,7 +1121,37 @@ resource "aws_elasticsearch_domain" "test" {
 `, rName, zoneAwarenessEnabled)
 }
 
-func testAccESDomainConfigWarm(rName, warmType string, enabled bool, warmCnt int) string {
+func testAccESDomainConfigWarmEnabled(rName, warmType string, warmCnt int) string {
```

Comment:
I think this can changed to be a variable in the original config instead of a separate one.